site/src/pages/TemplatePage/DAUCharts.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { render } from "testHelpers/renderHelpers"
+import { DAUChart, Language } from "./DAUCharts"
+
+import { screen } from "@testing-library/react"
+import { ResizeObserver } from "resize-observer"
+
+Object.defineProperty(window, "ResizeObserver", {
+  value: ResizeObserver,
+})
```

Comment:
Why are we adding this to the `window`? Feels like a codesmell but I blame my lack of context and understanding here.

Reply:
If this is needed, could we add it in a `beforeEach` or `beforeAll` block inside your `describe` block?

We also need to remove it from the `window` or it could affect other tests so it should be removed in a teardown function like `afterAll`.

Reply:
Unfortunately, there's no way to do this w/o modifying the global state. Since jest runs parallel, even if we use beforeAll I believe we may have state issues.